pkg/services/ngalert/notifier/channels/email.go

```diff
@@ -55,6 +56,7 @@ func NewEmailNotifier(model *NotificationChannelConfig, t *template.Template) (*
 		Message:     model.Settings.Get("message").MustString(),
 		log:         log.New("alerting.notifier.email"),
 		tmpl:        t,
+		bus:         bus.ProvideBus(),
```

Comment:
Maybe it's a good idea to inject this, passing a `bus.Bus` to the factory function. This can improve testability.

Reply:
Bus is no longer used in ngalerting at all, so there is no longer anything to inject.